guides/md/sequential_model.md

```diff
@@ -261,6 +261,7 @@ model.summary()
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:Please add `keras.layers.InputLayer` instead of `keras.Input` to Sequential model. `keras.Input` is intended to be used by Functional model.
```

Comment:
It looks like that `Input` should be an `InputLayer` would you mind fixing it at the same time?

Reply:
The warning will be gone in 2.6, so alternatively you could just deleted the warning by hand.

Reply:
As said, I manually delete it, check please!